.golangci.yaml

```
@@ -29,7 +29,7 @@ linters:
     - exhaustive
   disable:
     - unused
-    - errcheck  # Temporarily disabled due to too many warnings
+    #- errcheck  # Temporarily disabled due to too many warnings
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)